### PR TITLE
Q1 Strategy - move words

### PR DESCRIPTION
**Pull request by smellems**
Move Improve before Learn
Move Goals before Actions

### CODE REVIEW

**Comment by myp192** [comment]
That works for me 👍

File: _pages/en/strategy-learning-automating-improving.md [hunk 1]
```diff
@@ -1,30 +1,56 @@
...
 IITB teams should:
 
-#### Use up to 20% of time to learn and improve
+#### Use up to 20% of time to improve and learn
 
 **For all the following and other related activities, use CATS codes for continuous improvement (######).**
 
+##### Improve
+
+- Understand teams performance levels and set realistic goals
```
Comment (jaysonmc) [inline _pages/en/strategy-learning-automating-improving.md]: Im not sure what this line means ...

File: _pages/en/strategy-learning-automating-improving.md [hunk 2]
```diff
@@ -1,30 +1,56 @@
...
+- Identify gaps in your daily work performance and establish improvement priorities
+- Gather metrics (ex.: start and end date) for things your team works on
+- Show improvement based on metrics, the [Goals](#goals) and your team's mandate
+- Review existing processes and governance structure
+- Identify issues and raise them to your manager or as Grassroots initiatives or Proof of concepts
```
Comment (jaysonmc) [inline _pages/en/strategy-learning-automating-improving.md]: Im off network atm, but should link to grassroots inititaive and how one would launch a PoC

File: _pages/en/strategy-learning-automating-improving.md [hunk 3]
```diff
@@ -1,30 +1,56 @@
...
+- Automate repetitive manual tasks where possible, at the team and individual level
```
Comment (jaysonmc) [inline _pages/en/strategy-learning-automating-improving.md]: If I don't know how to automate, where should I start?

File: _pages/en/strategy-learning-automating-improving.md [hunk 4]
```diff
@@ -1,30 +1,56 @@
...
+
 ##### Learn
 
 - Stay up to date on trends and technologies inside and outside the GC
```
Comment (jaysonmc) [inline _pages/en/strategy-learning-automating-improving.md]: How?

**Pull request merged**